index.html

```diff
@@ -0,0 +1,268 @@
+       <p class="section-par text-color margin-24">I can help you build a product , feature or website Look through some of my work and experience! If you like what you
+    see and have a project you need coded,
+    don’t hestiate to contact me.</p>
+    
+        <div class="image-0">
+          <img src="socialmedia/design.png" alt="image">
+        </div>
+
+   </section>
+
+
+   <section class="section-2">
+     <div class="gridd">
+      <div class="my-recent-works">
+       <div class="H1">
+         <h1 > MY RECENT WORK</h1>
+       </div>
+      <div class="line">
+      <img  src="image/line.png" alt="line">
+      </div>
+      <div class="main-image">
+        <img src="image/image1.png" alt="main image">
+      </div>
+      <div class="H2">
+        <h2 > Multi-post Stories</h2>
+      </div>
+       <div class="p2">
+        <p>A daily selection of privately personalized reads;
+         no accounts or sign-ups required. has been the industry's standard
+         dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+       </div>
+       <div>
+      <ul class="buttons-0">
+        <li> <button class="btn" type="button">CSS</button></li>
+        <li> <button class="btn" type="button">HTML</button></li>
+        <li> <button  class="btn" type="button">Bootstrap</button></li>
+        <li> <button class="btn" type="button">Ruby</button></li>
+      </ul>
+      </div>
+      <div class="see-project">
+        <a class="seep" href="#"> see project</a>
+      </div>
+      </div>
+ 
+
+
+
+      <div class="profesional-art-1">
+        <div class="H3">
+        <h2> Profesional Art Printing Data</h2>
+        </div>
+        <div class="p2">
+        <p>A daily selection of privately personalized reads;
+          no accounts or sign-ups required.
+          has been the industry's standard</p>
+          </div>
+        <div class="button-0">
+          <ul class="buttons-0">
+            <li> <button class="btn" type="button">HTML</button></li>
+            <li> <button class="btn" type="button">Bootstrap</button></li>
+            <li> <button class="btn" type="button">Ruby</button></li>
+          </ul>
+        </div>
+        <div class="see-project-1">
+          <a class="project" href="#"> see project</a>
+        </div>
+        </div>
+
+<div class="profesional-art-1">
+  <div class="H3">
+    <h2> Profesional Art Printing Data</h2>
+  </div>
+  <div class="p2">
+    <p>A daily selection of privately personalized reads;
+      no accounts or sign-ups required.
+      has been the industry's standard</p>
+  </div>
+  <div class="button-0">
+    <ul  class="buttons-0">
+      <li> <button class="btn" type="button">HTML</button></li>
+      <li> <button class="btn" type="button">Bootstrap</button></li>
+      <li> <button class="btn" type="button">Ruby</button></li>
+    </ul>
+  </div>
+  <div class="see-project-1">
+    <a class="project" href="#"> see project</a>
+  </div>
```

Comment:
- [ ] Kindly make your button take the whole width, to match the Figma design.
- [ ] Kindly add some padding to your button here, to match the Figma design.
- [ ] Kindly make the text inside the button white, to match the Figma design.
- [ ] Kindly add an effect on the button when the user hovers or clicks on it as the Figma design says.

Your design.
![image](https://user-images.githubusercontent.com/35535130/166127985-357c33db-79d2-4b68-bfd9-f673171af91e.png)

Figma design
![image](https://user-images.githubusercontent.com/35535130/166128009-e6cf02f8-8808-4ad6-9be6-83b087d477a5.png)